### PR TITLE
Updated snapshot manager driver help to include new features

### PR DESCRIPTION
**Pull request by EmirVarupa**

### CODE REVIEW

File: connector/doc/NHK_Snapshot_Manager.md [hunk 1]
```diff
@@ -53,8 +60,18 @@ Via **Delete Debug Element**, you can remove a created debug element.
...
+### Debug Connector Configuration
 
-If you upload a debug connector using the protocol debug tools, make sure to append `-debug` and select "Remove Sequence" (available from version 1.0.0.6 onwards).
+This page is used to configure the creation of the debug connector. You can also directly create a debug connector here.
 
-Via **Delete Debug Protocol Version**, you can remove a debug protocol version.
+- **Add Write Parameters**: If this option is enabled, the debug protocol will be modified to include write parameters for all read parameters in the protocol.
+- **Mark Parameters as Saved**: If this option is enabled, the debug protocol will be modified to mark all parameters as saved.
+- **Connector**: This dropdown lists all the connectors that a debug connector can be created for.
+- **Connector Version Extension**: This parameter specifies the debug connector version. The debug connector version will be appended with `-debug`.
```
Comment (MariekeGO) [inline connector/doc/NHK_Snapshot_Manager.md:76]: Is this parameter name correct? It seems odd that the parameter specifying the version, to which the extension will be appended, should itself be called "Connector Version Extension".

Comment (MariekeGO) [reply connector/doc/NHK_Snapshot_Manager.md:76]: This means that this parameter is present both on the General page and the Debug Connector Configuration page – is this correct?
Based on the explanation higher up, I'd rephrase this as follows then:

```suggestion
- **Connector Version Extension**: This parameter specifies the debug connector version extension, which will be appended to the debug connector version.
```

Comment (EmirVarupa) [reply connector/doc/NHK_Snapshot_Manager.md:76]: Yes, the parameter is present in both pages.

I think it would be a good idea to let the user know what the default value of this parameter is:
```suggestion
- **Connector Version Extension**: This parameter specifies the debug connector version extension. By default, it is set to `debug`, and it will be appended to the debug connector version.
```

**Review by MariekeGO — APPROVED** [review]
